### PR TITLE
[csrng/dv] Initial dv environment

### PR DESCRIPTION
**Pull request by senelson7**
Signed-off-by: Steve Nelson <steve.nelson@wdc.com>

### CODE REVIEW

**Review by mwbranstad — APPROVED** [review]
Appears that the sanity test is only reading the CNTL register for a POR of zero - LGTM.

**Comment by senelson7** [comment]
Yep, there's a TODO in the sanity test. Just does a simple register read for now to bring up the initial dv environment.

File: hw/ip/entropy_src/dv/env/entropy_src_env.core [hunk 1]
```diff
@@ -5,6 +5,9 @@ CAPI=2:
 name: "lowrisc:dv:entropy_src_env:0.1"
 description: "ENTROPY_SRC DV UVM environment"
 filesets:
+  files_rtl:
+    depend:
+        - lowrisc:ip:entropy_src:0.1
```
Comment (weicaiyang) [inline hw/ip/entropy_src/dv/env/entropy_src_env.core:10]: this and line 41 seem not relevant and not needed

**Review by weicaiyang — APPROVED** [review]
LGTM with a small comment

File: hw/ip/entropy_src/dv/env/entropy_src_env.core (hunk 1 above)
Comment (senelson7) [reply hw/ip/entropy_src/dv/env/entropy_src_env.core]: Shouldn't have been any changes to an entropy_src file so I removed it from this PR

File: hw/ip/csrng/dv/csrng_sim_cfg.hjson [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+                // Common CIP test lists
+                "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/mem_tests.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/intr_test.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/tl_access_tests.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/stress_tests.hjson"]
+
+  // Add additional tops for simulation.
+  sim_tops: ["-top {name}_bind"]
```
Comment (sriyerg) [inline hw/ip/csrng/dv/csrng_sim_cfg.hjson:38]: This should be 
```suggestion
  sim_tops: ["-top csrng_bind"]
```
Seems like a bug in `util/uvmdvgen/sim_cfg.hjson.tpl` - should be `${name}`. Would you please fix that as well?

Comment (senelson7) [reply hw/ip/csrng/dv/csrng_sim_cfg.hjson]: Done

**Review by sriyerg — APPROVED** [review]

**Pull request merged**